www/src/pages/table/forms/TransformationForm.tsx

```diff
@@ -0,0 +1,284 @@
+import { WithStyles, withStyles } from "@material-ui/styles";
+import {
+  Button,
+  Radio,
+  Tag,
+  Space,
+  Table,
+  Modal,
+  Form,
+  Col,
+  Row,
+  Select,
+  InputNumber,
+} from "antd";
+import { observer } from "mobx-react";
+import { ActionType } from "@ant-design/pro-table";
+import { useRef, useState } from "react";
+import Editor from "@monaco-editor/react";
+import { TransformationTable, useStores } from "../../../models";
+import {
+  postData,
+  TPayload,
+} from "../../../models/transformationTable/TransformationStore";
+
+const styles = {
+  table: {
+    "& div.ant-table-container": {
+      border: "1px solid #bbb",
+      borderRadius: 4,
+      borderLeft: "1px solid #bbb !important",
+    },
+    "& div.ant-card-body": {
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    "& th": {
+      fontWeight: 600,
+    },
+  },
+};
```

Comment:
redundant - or move inline styles to here

Reply:
removed the styles